### DISCUSSION TITLE
Can`t start vmare workstation

### DISCUSSION THREAD

**Discussion by TwelveNight** [hyprwm/Hyprland#3665] [Q&A]
I just change DE(x11) to WM(wayland) that use hyprland,but I can`t start my vmware-workstation on arch linux
my vmware verstion:VMware Workstation 17.0.1 build-21139696
The error is here,
![image](https://github.com/hyprwm/Hyprland/assets/113519679/5e5b1aae-a76d-462d-ac1d-bd662906fda2)
I just change the display 1 to :0, but is still the same error

**Comment by MightyPlaza** [comment]
do you have xwayland support? (hyprctl version)

**Reply by TwelveNight** [reply]
yes, seems I not settting the  `GDK_BACKEND=x11`
